libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -792,7 +792,7 @@ AP_AHRS_DCM::drift_correction(float deltat)
     // flat, but still allow for yaw correction using the
     // accelerometers at high roll angles as long as we have a GPS
     if (AP_AHRS_DCM::use_compass()) {
-        if (have_gps() && is_equal(gps_gain,1.0f)) {
+        if (have_gps() && is_equal<float>(gps_gain,1.0f)) {
```

Comment:
Like I said, the code isn't wrong, it's just using different ways to do the same thing. But I believe we should maintain a consistent way of coding, if we do it one way in one place we should do it everywhere - specially for things in the same PR.